hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
```

Comment:
Since they are also made as default, do we need overrides here ?

Reply:
@rbair23 asked us to make sure `application.properties` matched the mainnet file `0.0.121` contents, but we also made it the default.

@rbair23 Can we remove this from `0.0.121` since it is now the default in the `bootstrap.properties` shipped in the JAR file?

Reply:
Got it. Thanks. Doesn't hurt, was just curious why we need.